### PR TITLE
Spike multiple SQL instances

### PR DESCRIPTION
**Pull request by NickGrantX**
# Motivation and Context
Large Action Rules were affecting the DB performance - making it non-responsive - which would affect time-critical message processing.

# What has changed
Added config for Action DB. Modified DB monitor script to use Action DB. Added scripts for connecting to Action DB.

# How to test?
Terraform a new environment (prod-like) using this branch. Use the kubernates branch of the same name.

Use `dosqla` to connect to Action DB. Use `dosql-i-know-what-i-am-doing-action` to connect in read-write mode. Check the print batch script.

# Links
Trello: https://trello.com/c/fAmdlRkN

### CODE REVIEW

**Comment by NickGrantX** [comment]
[Move Case & Action onto different Cloud SQL instances (3 days)](https://trello.com/c/fAmdlRkN/760-move-case-action-onto-different-cloud-sql-instances-3-days)

File: i_know_what_i_am_doing_action.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+setterm -term linux -fore red
+echo -e '\n'
+echo '▄▄▌ ▐ ▄▌▄▄▄ .     ▄▄▄· ▄▄▄  ▄▄▄ .    ·▄▄▄▄              • ▌ ▄ ·. ▄▄▄ .·▄▄▄▄      • ▌ ▄ ·.       ·▄▄▄▄  ▄▄▄ .'
+echo '██· █▌▐█▀▄.▀·    ▐█ ▀█ ▀▄ █·▀▄.▀·    ██▪ ██ ▪     ▪     ·██ ▐███▪▀▄.▀·██▪ ██     ·██ ▐███▪▪     ██▪ ██ ▀▄.▀·'
+echo '██▪▐█▐▐▌▐▀▀▪▄    ▄█▀▀█ ▐▀▀▄ ▐▀▀▪▄    ▐█· ▐█▌ ▄█▀▄  ▄█▀▄ ▐█ ▌▐▌▐█·▐▀▀▪▄▐█· ▐█▌    ▐█ ▌▐▌▐█· ▄█▀▄ ▐█· ▐█▌▐▀▀▪▄'
+echo '▐█▌██▐█▌▐█▄▄▌    ▐█ ▪▐▌▐█•█▌▐█▄▄▌    ██. ██ ▐█▌.▐▌▐█▌.▐▌██ ██▌▐█▌▐█▄▄▌██. ██     ██ ██▌▐█▌▐█▌.▐▌██. ██ ▐█▄▄▌'
+echo ' ▀▀▀▀ ▀▪ ▀▀▀      ▀  ▀ .▀  ▀ ▀▀▀     ▀▀▀▀▀•  ▀█▄▀▪ ▀█▄▀▪▀▀  █▪▀▀▀ ▀▀▀ ▀▀▀▀▀•     ▀▀  █▪▀▀▀ ▀█▄▀▪▀▀▀▀▀•  ▀▀▀ '
+echo "You better know what you're doing, buddy."
+echo 'Godspeed...'
+echo
```
Comment (AdamHawtin) [inline i_know_what_i_am_doing_action.sh]: Would be helpful if this said something about being in the action DB specifically.

File: utilities/db_helper.py [hunk 2]
```diff
@@ -29,9 +29,9 @@ def execute_sql_query_with_write(cursor, sql_query, values: tuple):
 
 
 @contextlib.contextmanager
-def open_write_cursor():
-    conn = psycopg2.connect(f"dbname='{Config.DB_NAME}' user='{Config.DB_USERNAME}' host='{Config.DB_HOST_WRITE}' "
-                            f"password='{Config.DB_PASSWORD}' port='{Config.DB_PORT_WRITE}'{Config.DB_USESSL}")
+def open_write_cursor(db_host=Config.DB_HOST, extra_options=""):
```
Comment (AdamHawtin) [inline utilities/db_helper.py:32]: This one needs to default to `Config.DB_HOST_WRITE` instead, it needs to connect to the write instance.

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

File: utilities/db_helper.py (hunk 2 above)
Comment (NickGrantX) [reply utilities/db_helper.py:32]: `open_write_cursor` is only used by the print batch thingy and it overrides the default to be the action DB

**Review by AdamHawtin — APPROVED** [review]

**Review by liamtoozer — APPROVED** [review]

**Pull request merged**